src/jquery.ime.inputmethods.js

```diff
@@ -1582,6 +1586,10 @@
 			autonym: 'Svenska',
 			inputmethods: [ 'sv-normforms' ]
 		},
+		szl: {
+			autonym: 'Ślōnskŏ',
```

Comment:
The code and the tests are OK, thanks! However, I'm wondering about the name of the language. The name in the interlanguage links list on Wikipedia appears as "Ślůnski". I _guess_ that this is related to the different orthographies that the language uses. I don't have a preference for a particular orthography, but the name should be written consistently everywhere in the interface of the wiki sites. And when it's relevant, it should also be consistent with other websites in this language.

If the name "Ślōnskŏ" is more common today, it can be used here, and I can also update it in other places MediaWiki. However, before I do that, I would love to see examples of this name's usage on other websites or in books (in the spirit of "reliable sources" on Wikipedia).

Reply:
Your guess is absolutely right, it is because of more orthographies. We allow both orthographies on Wikipedia (and I included both on the keyboard mapping), but the "Ślōnskŏ" is going to become the mainstream. In order not to complicate your job, I replaced it in a new commit with "Ślůnski". However, in the future most probably the Wikipedia orthography will be standardized to "Ślōnskŏ"-form and then I can open a ticket to change it on MediaWiki. The orthograpies are called ślabikŏrzowy szrajbōnek and steuerowy szrajbůnek. Thank You!